src/types.ts

```diff
@@ -3527,7 +3552,7 @@ export enum HostEvent {
     /**
      * Trigger the **Save**  action on a Liveboard or Answer.
      * Saves the changes.
-     * @param - `vizId` is required in Spotter Embed.
+     * @param - `vizId` refers to the answer id in spotter Embed, it is required in spotter Embed.
```

Comment:
`vizId` refers to the Answer ID in Spotter Embed and is required in Spotter Embed.